### DISCUSSION TITLE
Whats the highest bitrate I can get?

### DISCUSSION THREAD

**Discussion by CubiiGit** [Tyrrrz/YoutubeExplode#620] [Q&A]
I want to make an only-music downloader and want - logically - the best quality. When I check the files, I see that they are only 128 kbits? Is this the highest I can get?

**Comment by Tyrrrz** [comment]
Yeah it is

**Reply by CubiiGit** [reply]
Oh, ok. I guess it's limited due to YouTube restrictions, right?

**Reply by Tyrrrz** [reply]
Yes, uploaded audio is recoded to this bitrate
